libs/design-system/src/lib/Pages/NetworkDetailPage/index.tsx

```diff
@@ -39,11 +38,15 @@ const NetworkDetailPage = () => {
         <div className="relative">
           <a
             href="/network"
-            className="dark:text-primary-blue-dark absolute top-[110px] right-0 flex max-w-fit text-primary-blue md:top-0 md:left-0"
+            className="absolute top-[110px] right-0 flex max-w-fit text-primary-blue hover:opacity-75 dark:text-blue-dark md:right-auto md:left-0 md:top-0 md:left-0 md:py-6"
           >
             <ChevronLeftIcon /> Network
           </a>
-          <TabMenu tabs={tabs} onTabChange={setSelectedNetworkIndex} />
+          <TabMenu
+            tabs={tabs}
+            onTabChange={setSelectedNetworkIndex}
+            centered={true}
```

Comment:
```suggestion
            centered
```

Reply:
Thanks! On the network pages? Yeah, we'll have to remove the footers from the page components